plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/operations/CreateFeatureAboveOperation.java

```diff
@@ -96,12 +104,85 @@ protected FeatureIDEEvent operation() {
 			featureModel.addFeature(newCompound);
 			featureModel.getStructure().setRoot(newCompound.getStructure());
 		}
+
+		if (newGraphicalFeature.getGraphicalModel().getLayout().getLayoutAlgorithm() == 0) {
+			adjustPositions();
+		}
+
 		return new FeatureIDEEvent(featureModel, EventType.FEATURE_ADD_ABOVE, parent != null ? parent.getFeature() : null, newCompound);
 	}
 
+	/**
+	 * adjusts position of the new feature and its children in manual layout
+	 */
+	private void adjustPositions() {
```

Comment:
Declare const with descriptive names instead of random integer values (Magic Number Anti-Pattern)